.github/workflows/python_tests.yml

```diff
@@ -43,9 +43,8 @@ jobs:
 
       - name: Run tests with caching
         run: |
-          cd cairo
           uv run compile_os
-          uv run pytest tests -n logical --durations=0 -v
+          uv run pytest cairo/tests -n logical --durations=0 -v -s --log-cli-level=DEBUG
```

Comment:
what is debug for ?

Reply:
Just more logs ; I can remove but I’ve guessed that it costs nothing to l eave it and gives more info on ci jobs which are sometimes hard to understand